plugins/polio/js/src/domains/VaccineModule/SupplyChain/Details/SupplyChainTabs.tsx

```diff
@@ -32,11 +32,22 @@ export const SupplyChainTabs: FunctionComponent<Props> = ({
             value={tab}
             classes={{
                 root: classes.tabs,
-                indicator: classes.indicator,
+                // indicator: classes.indicator,
```

Comment:
I'm wondering if there is a reason why you're often commenting instead of just removing the lines?

Reply:
I do it during dev. I find it easier to revert than going through git or spamming "undo" and it sometimes helps when I'm refactoring to still see the code I'm coming from. But then I often forget to delete the comments when I'm done